src/role_scout.js

```diff
@@ -115,11 +115,11 @@ roles.scout.execute = function(creep) {
           delete creep.memory.last; // Delete to reset stuckness.
         }
       }
-      if (search.path.length > 0) {
-        creep.move(creep.pos.getDirectionTo(search.path[0]));
-      } else {
-        creep.moveTo(targetPosObject);
-      }
+      //if (search.path.length > 0) {
+      //creep.move(creep.pos.getDirectionTo(search.path[0]));
+      //} else {
+      creep.moveTo(targetPosObject);
+      //}
```

Comment:
So we cant chekc for path length, we have to check for last path position = target?